packages/tizen_rpc_port/example/server/lib/message_server.dart

```diff
@@ -166,7 +166,7 @@ class Message extends StubBase {
 
   Future<void> _onUnregisterMethod(
       ServiceBase service, Port port, Parcel parcel) async {
-    service.onUnregister();
+    await service.onUnregister();
```

Comment:
Note that this file is auto-generated by tidlc. The change should be reflected to the code generator.

Reply:
I couldn't fix this without modifying tidlc's code directly. (When I tested latest tidlc code, only some comments were added.) so I added analysis_options.yaml to example/server,client. And i had disabled `unawaited_futures` rule.